tiledb/sm/subarray/subarray.cc

```diff
@@ -307,8 +307,9 @@ Status Subarray::add_range(
   if (oob_warning.has_value())
     LOG_WARN(oob_warning.value() + " on dimension '" + dim_name + "'");
 
-  // Update is default.
+  // Update is default and stats counter
   is_default_[dim_idx] = range_subset_[dim_idx].is_implicitly_initialized();
+  stats_->add_counter("add_range", 1);
```

Comment:
I think we need one counter per dimension here. stats_->add_counter("add_range_dim_" + std::to_string(dim_idx), 1);